mantle/platform/machine/unprivqemu/cluster.go

```diff
@@ -104,9 +105,13 @@ func (qc *Cluster) NewMachineWithOptions(userdata *conf.UserData, options platfo
 		Size:        qc.flight.opts.DiskSize,
 	}
 
-	builder.AddPrimaryDisk(&primaryDisk)
+	if err = builder.AddPrimaryDisk(&primaryDisk); err != nil {
+		return nil, errors.Wrapf(err, "failed adding primary disk")
```

Comment:
Nonblocking, but I usually omit "failed" and "error" from these messages because usually the toplevel emits an `error:` prefix to the thing, so it's redundant.

If each wrapper did it you can end up with e.g. `error: failed foo: error unlinking` instead of `error: foo: unlinking` - and the latter to me just reads better.

I think of these strings as like "interesting function stack frame descriptions".

Reply:
Yup, agreed!